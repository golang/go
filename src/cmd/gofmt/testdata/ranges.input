//gofmt -s

// Test cases for range simplification.
package p

func _() {
	for a, b = range x {}
	for a, _ = range x {}
	for _, b = range x {}
	for _, _ = range x {}

	for a = range x {}
	for _ = range x {}

	for a, b := range x {}
	for a, _ := range x {}
	for _, b := range x {}

	for a := range x {}
}
