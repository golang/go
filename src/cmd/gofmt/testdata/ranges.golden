//gofmt -s

// Test cases for range simplification.
package p

func _() {
	for a, b = range x {
	}
	for a = range x {
	}
	for _, b = range x {
	}
	for range x {
	}

	for a = range x {
	}
	for range x {
	}

	for a, b := range x {
	}
	for a := range x {
	}
	for _, b := range x {
	}

	for a := range x {
	}
}
